storage/range.go

```diff
@@ -915,6 +904,7 @@ func (r *Range) maybeGossipFirstRange() error {
 	if ok, err := r.getLeaseForGossip(); !ok || err != nil {
 		return err
 	}
+	log.Infof("gossiping first range from store %d, range %d", r.rm.StoreID(), r.Desc().RaftID)
```

Comment:
Infof is what it was when it lived inside store.go. I think it's important enough and only being logged from one node in the cluster. I'd rather leave it in.